trade-finance-manager-ui/server/api-response-types/amendments.ts

```diff
@@ -0,0 +1,10 @@
+type ErrorResponse = {
```

Comment:
I think you should be able to import `ApiErrorResponseBody` from `@ukef/dtfs2-common` for this type (although that type uses `message` instead of `data` - probably better to ask @AlexBramhill here if we want to start using that type more to make the way we handle api errors more consistent)

Reply:
Would this be with the `ApiError` class? (I can't find `ApiErrorResponseBody`). I think I'd probably be keen to not change it here, as it would require refactoring all the other amendments error types alongside it (or otherwise e.g. reverting back from the newly commonised tests)

Reply:
If this is the existing amendments implementation, I think for now well have to keep it with `data` for now and not commonise

Reply:
Sounds good to me